pyproject.toml

```diff
@@ -53,13 +53,24 @@ docs = [
     "nbsphinx",
     "sphinx-autodoc-typehints",
     "sphinx-autoapi",
-    "qiskit-terra[visualization]"
+    "qiskit-terra[visualization]",
+    "colorama>=0.4.4",
+    "tabulate>=0.8.9"
```

Comment:
I'm not sure about the cause, because frankly, it's kind of hard to debug jupyter notebooks.
However, if I don't add colorama and tabulate (which is implicitly needed for `df.to_markdown()`), the docs build will fail with a `ModuleNotFoundError`.
On the other hand, I'm not sure if we need `mqt.core[evaluation]`, since the point was to separate the evaluation module, right?
I suggest that we keep tabulate, and remove colorama since per the other suggestion I should remove it from the project anyway.